android/src/main/java/com/onegini/mobile/sdk/reactnative/managers/AuthenticatorManager.kt

```diff
@@ -18,120 +18,95 @@ import javax.inject.Singleton
-                override fun onSuccess() {
-                    callback.onSuccess()
-                }
-
-                override fun onError(error: OneginiAuthenticatorDeregistrationError) {
-                    callback.onError(error.errorType.toString(), error.message)
-                }
-            })
-    }
-
-    fun getUserProfile(profileId: String?): UserProfile? {
-        if (profileId == null) {
-            return null
-        }
-        oneginiSDK.oneginiClient.userClient.userProfiles.forEach {
-            if (it.profileId == profileId) {
-                return it
-            }
-        }
-        return null
-    }
-
-    fun getNotRegisteredAuthenticators(profile: UserProfile, type: Int): OneginiAuthenticator? {
-        val notRegisteredAuthenticators: Set<OneginiAuthenticator> = oneginiSDK.oneginiClient.userClient.getNotRegisteredAuthenticators(profile)
-        for (auth in notRegisteredAuthenticators) {
-            if (auth.type == type) {
-                return auth
-            }
-        }
-        return null
-    }
-
-    fun getRegisteredAuthenticators(profile: UserProfile, type: Int): OneginiAuthenticator? {
-        val registeredAuthenticators: Set<OneginiAuthenticator> = oneginiSDK.oneginiClient.userClient.getRegisteredAuthenticators(profile)
-        for (auth in registeredAuthenticators) {
-            if (auth.type == type) {
-                return auth
-            }
+        override fun onError(error: OneginiAuthenticatorDeregistrationError) {
+          callback.onError(error.errorType.toString(), error.message)
         }
-        return null
-    }
-
-    fun getRegisteredAuthenticator(profile: UserProfile, id: String): OneginiAuthenticator? {
-        return oneginiSDK.oneginiClient.userClient
-            .getRegisteredAuthenticators(profile)
-            .firstOrNull { it.id == id }
-    }
-
-    fun getAuthenticator(profile: UserProfile, id: String): OneginiAuthenticator? {
-        return oneginiSDK.oneginiClient.userClient
-            .getAllAuthenticators(profile)
-            .firstOrNull { it.id == id }
-    }
-
-    fun setPreferredAuthenticator(authenticatorId: String) {
-        val userProfile = oneginiSDK.oneginiClient.userClient.authenticatedUserProfile
-            ?: throw OneginiReactNativeException(NO_PROFILE_AUTHENTICATED)
-
-        val authenticator = getRegisteredAuthenticator(userProfile, authenticatorId)
-            ?: throw OneginiReactNativeException(AUTHENTICATOR_DOES_NOT_EXIST)
-
-        oneginiSDK.oneginiClient.userClient.setPreferredAuthenticator(authenticator)
-    }
-
-    interface RegistrationCallback {
-        fun onSuccess(customInfo: CustomInfo?)
-        fun onError(code: String?, message: String?)
-    }
-
-    interface DeregistrationCallback {
-        fun onSuccess()
-        fun onError(code: String?, message: String?)
-    }
+      })
+  }
+
+  fun getUserProfile(profileId: String?): UserProfile? {
```

Comment:
When I edited code, I've formatted it with onewelcome code style and that's why there's so many changes in this file.